source/index.html.md.erb

```diff
@@ -7,13 +7,14 @@ weight: 0
 
 GOV.UK Frontend provides the elements developers need to create consistent, easy to use, fast and secure government services.
 
+<%=warning_text('If your service still uses GOV.UK Template, GOV.UK Frontend Toolkit or GOV.UK Elements, you should [install the latest v4 release of GOV.UK Frontend](../v4/installing-with-npm/) and follow the [guidance on migrating from our old frameworks](../v4/migrating-from-legacy-products/).') %>
```

Comment:
Unfortunately we can't use Markdown here – this'll need to be HTML, I think

<img width="758" height="151" alt="Screenshot 2025-08-28 at 16 30 43" src="https://github.com/user-attachments/assets/41c5604f-d6fd-402e-8fee-1fc638325cf3" />

Reply:
Done